ports/psoc6/freeze/vfs_lfs2.py

```diff
@@ -22,6 +22,10 @@
     vfs = os.VfsLfs2(bdev, progsize=write_size, readsize=read_size)
     os.mount(vfs, "/")
 
-print("Internal LFS2 filesystem mounted at /\n")
+if "QSPI_Flash" in dir(psoc6):
```

Comment:
Why this conditional?

Reply:
Because if you notice the previous print statement, it says Internal flash fs is mounted always even when external flash was mounted. So I am just printing based on the condition which flash was mounted.

Reply:
ok ok, i did  not see the Internal´vs external. Reading too quick :)